igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/client/model/v3/V3Log.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2018 Schibsted ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.travis.client.model.v3;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class V3Log {
+  private Integer id;
+  private String content;
+  @JsonProperty("log_parts")
+  private List<V3LogPart> logParts;
+
+  public Integer getId() {
+    return id;
+  }
+
+  public void setId(Integer id) {
+    this.id = id;
+  }
+
+  public String getContent() {
```

Comment:
Turns out that the `content` field isn't yet populated right after a build completes. Instead, the log is split over several `logParts` fields. If you do the same API call a few minutes later, these `logParts` will have been concatenated and put into the `content` field.
With this commit, we support both ways.